core/config.h

```diff
@@ -46,6 +46,9 @@ d_r_config_init(void);
 void
 d_r_config_exit(void);
 
+bool
```

Comment:
nit: do we need this function in header as opposed to making it static? I don't see it being used outside of `config.c`.

Reply:
unix/os.c has `ASSERT(d_r_config_initialized())`